{
  ModelNode op=getOperation("runtimeException","attr1",5,"good");
  op.get(OPERATION_HEADERS,ROLLBACK_ON_RUNTIME_FAILURE).set(false);
  ModelNode result=controller.execute(op,null,null,null);
  assertEquals(FAILED,result.get(OUTCOME).asString());
  assertTrue(result.get("failure-description").toString().indexOf("runtime exception") > -1);
  assertFalse(sharedState.get());
  result=controller.execute(getOperation("good","attr1",1),null,null,null);
  System.out.println(result);
  assertEquals("success",result.get("outcome").asString());
  assertEquals(5,result.get("result").asInt());
}
